### ISSUE TITLE
Routes should match XHR's opened with lowercase methods

### ISSUE THREAD

**Issue by brian-mann** [cypress-io/cypress#607]
This currently fails:

```coffeescript
cy
  .route(/foo/, {}).as("getFoo")
  .window().then (win) ->
    xhr = new win.XMLHttpRequest
    xhr.open("get", "/foo")
    xhr.send()
  .wait("@getFoo")
```

![screen shot 2017-08-26 at 4 34 57 pm](https://user-images.githubusercontent.com/1268976/29744924-b2f7eb6a-8a7c-11e7-9b9b-0c78b23b8c76.png)

This is happening because the method "GET" does not strictly equal "get".

We need to automatically match XHR's opened with lowercase methods.

**Comment by brian-mann** [comment]
The code for this is done, but this has yet to be released. We'll update the issue and reference the changelog when it's released.

**Comment by brian-mann** [comment]
Fixed in [`0.20.0`](https://docs.cypress.io/guides/references/changelog.html#0-20-0)
